### DISCUSSION TITLE
Give *HTML* file instead of *index.qmd* in the *_quarto.yml* for books, website, blog

### DISCUSSION THREAD

**Discussion by FannyVille** [quarto-dev/quarto-cli#6057] [Feature Requests]
### **Request:**
Would it be possible one day to give an *.html* file instead of *index.qmd* for blog, website or book in the *_quarto.yml* ?

### **issues:**
In the case of a large number of HTML files already generated via Rmarkdown or Quarto in other projects created with large databases, it is annoying to have to transfer the data and the *.qmd* document to the blog, website or book project...
It would be very interesting to just copy the HTML files and be able to refer to them in the *_quarto.yml.* 

**If this isn't possible, what strategy should be put in place?**

### **Current solution:**
Following my analyses with quarto, I render the files in HTML format with the option ` embed-resources: true`, so I only have one HTML file without any document/attachment folder. 

 - I copy/paste the HTML file into the *posts* folder of my blog/website/book project.
 - I create the *index.qmd* file
 - I insert a code chunk in html format and use an iframe to embed my html file.

````{=html}
<iframe src="MY_FILE.html" scrolling="no" style="overflow:hidden" width="100%" height="7749.12px" />
````
This is not entirely appropriate because there are several problems:
- the toc of HTML files is no longer dynamic. When you scroll down in the file, the toc does not follow. 
- the body is reduced in width
- we end up with two scrollbars, one for the web page and one inside for the html document. 
    -> To solve this ugly issues, I use the iframe options: `scrolling="no" style="overflow:hidden"`. 
    -> But I have to give the height of my HTML file.

All this is manual and tedious, and it would be easier if we could simply refer to html files in the *_quarto.yml* rather than just index.qmd files.

I looked to see if anyone had already raised the issue. I think so but not exactly this one. 
I apologise in advance if the same query was already made...

Thank you

**Comment by mcanouil** [comment]
Thanks for the nice suggestion.
